subprojects/docs/src/docs/userguide/running-builds/introduction/build_scans.adoc

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[build_scans]]
+= Build Scans
+
+A build scan is a *representation of data captured* as you run your build.
+
+image::gradle-basic-8.png[]
+
+== Build Scans
+Gradle captures your build data and sends it to the link:https://scans.gradle.com/[Build Scan Service].
+The service then transforms the data into information you can use and share with others.
+
+image::build-scan-1.png[]
+
+The information that scans provide can be a big help when troubleshooting, collaborating on, or optimizing the performance of your builds.
+
+For example, with a build scan, it’s no longer necessary to copy and paste error messages or include all the details about your environment each time you want to ask a question on Stack Overflow, Slack, or the Gradle Forum.
+Instead, just copy the link to your latest build scan.
+
+image::build-scan-2.png[]
+
+== Enable Build Scans
+
+To enable build scans on a gradle command, add `--scan` to the command line option:
+
+[source]
+----
+ ./gradlew stageDocs --scans
```

Comment:
```suggestion
 ./gradlew compileJava --scans
```

`stageDocs` is a task specific to the gradle/gradle build. We should use a conventional task in the examples